src/hooks/UrqlUseQuery.re

```diff
@@ -24,12 +24,17 @@ type useQueryState('a) = {
 }
 type useQueryResponse('a) = (useQueryState('a), partialOperationContextFn);
 
-let useQueryResponseToRecord = (result: useQueryStateJs('a)) => {
-  let data = result->dataGet;
+let useQueryResponseToRecord =
+    (parse: Js.Json.t => 'response, result: useQueryStateJs('a)) => {
```

Comment:
I think the idea with `useQueryStateJs` is that the type argument here `'a` corresponds to the raw response from the GraphQL request _before_ it's been parsed by `graphql_ppx`. I've been using `'a`, `'b`, etc. with the thought that it's more OCaml-y, but I like what you did with `'response`, so perhaps I'll make these type signatures more descriptive in a follow up PR. This looks great though!